core/src/main/resources/application.properties

```diff
@@ -30,12 +30,19 @@
 
 #To configure Spring JPA properties to connect Klaw to internal file database
 
-spring.datasource.url=jdbc:h2:file:./klawprodb;DB_CLOSE_ON_EXIT=FALSE;DB_CLOSE_DELAY=-1;MODE=MySQL;CASE_INSENSITIVE_IDENTIFIERS=TRUE;
-spring.datasource.driver.class=org.h2.Driver
-spring.datasource.username=kafkauser
-spring.datasource.password=klaw
-spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.H2Dialect
-spring.h2.console.enabled=true
+#spring.datasource.url=jdbc:h2:file:./klawprodb;DB_CLOSE_ON_EXIT=FALSE;DB_CLOSE_DELAY=-1;MODE=MySQL;CASE_INSENSITIVE_IDENTIFIERS=TRUE;
```

Comment:
we might want to add commented mysql properties

Reply:
oh! these are not meant to be there! I will remove these ones